### PR TITLE
Lod dev

### PR DESCRIPTION
**Pull request by Kayvv**

### CODE REVIEW

File: src/cmlibs/exporter/webgl.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ def __init__(self, output_target=None, output_prefix=None):
         """
         super(ArgonSceneExporter, self).__init__("ArgonSceneExporterWebGL" if output_prefix is None else output_prefix)
         self._output_target = output_target
+        self.multiple_levels = False
+        self.tessellation_level = None
```
Comment (hsorby) [inline src/cmlibs/exporter/webgl.py:28]: These are private members and as such should start with an underscore `_`.

File: src/cmlibs/exporter/webgl.py [hunk 2]
```diff
@@ -38,6 +40,17 @@ def export(self, output_target=None):
...
+            self.tessellation_level = "medium"
+            self._document.setTessellationLevel("medium")
+            self.export_webgl()
+
+            self.tessellation_level = "high"
+            self._document.setTessellationLevel("high")
+            self.export_webgl()
+
+        self.tessellation_level = None
+        self._document.setTessellationLevel("low")
```
Comment (hsorby) [inline src/cmlibs/exporter/webgl.py:53]: If we are not specifying multiple tessellation levels then we won't be setting the tessellation level to `low`.

File: src/cmlibs/exporter/webgl.py [hunk 3]
```diff
@@ -130,7 +159,9 @@ def export_webgl_from_scene(self, scene, scene_filter=None):
 
         number_of_digits = math.floor(math.log10(number)) + 1
 
-        def _resource_filename(prefix, i_):
+        def _resource_filename(prefix, i_, tessellation_level=None):
+            if tessellation_level:
+                return f'{prefix}_{tessellation_level}_{str(i_).zfill(number_of_digits)}.json'
             return f'{prefix}_{str(i_).zfill(number_of_digits)}.json'
```
Comment (hsorby) [inline src/cmlibs/exporter/webgl.py:165]: This is a bit repeated the tessellation level should be parameterised better.

File: src/cmlibs/exporter/webgl.py [hunk 4]
```diff
@@ -61,6 +76,20 @@ def export_view(self):
...
         return f"{self._prefix}_{name}_view.json"
 
+    def _define_default_LOD_obj(self, index):
+        LOD_obj = {
+            "Preload": False,
+            "Levels": {
+                "medium": {
+                    "URL": f"{self._prefix}_medium_{index}.json"
+                },
+                "close": {
```
Comment (hsorby) [inline src/cmlibs/exporter/webgl.py:86]: Are `medium` and `close` specified in the standard for setting the level of detail or can they be any label?

**Review by hsorby — CHANGES_REQUESTED** [review]
Some changes are required here, and maybe a bit of help understanding some of the proposed changes.

File: src/cmlibs/exporter/webgl.py (hunk 4 above)
Comment (Kayvv) [reply src/cmlibs/exporter/webgl.py]: Alan Wu said: "Those are set and can't be any other label unless I modify them."

**Review by hsorby — APPROVED** [review]

**Pull request merged**